api/nodes/serializers.py

```diff
@@ -24,37 +24,14 @@ class NodeSerializer(JSONAPISerializer):
                                                'user and system. Any tag that a user will define in the UI will be '
                                                'a user tag')
 
-    links = LinksField({
-        'html': 'get_absolute_url',
-        'children': {
-            'related': Link('nodes:node-children', kwargs={'node_id': '<pk>'}),
-            'count': 'get_node_count',
-        },
-        'contributors': {
-            'related': Link('nodes:node-contributors', kwargs={'node_id': '<pk>'}),
-            'count': 'get_contrib_count',
-        },
-        'node_links': {
-            'related': Link('nodes:node-pointers', kwargs={'node_id': '<pk>'}),
-            'count': 'get_pointers_count',
-        },
-        'registrations': {
-            'related': Link('nodes:node-registrations', kwargs={'node_id': '<pk>'}),
-            'count': 'get_registration_count',
-        },
-        'files': {
-            'related': Link('nodes:node-files', kwargs={'node_id': '<pk>'})
-        },
-        'parent': {
-            'self': Link('nodes:node-detail', kwargs={'node_id': '<parent_id>'})
-        }
-    })
-    properties = ser.SerializerMethodField(help_text='A dictionary of read-only booleans: registration, collection,'
-                                                     'and dashboard. Collections are special nodes used by the Project '
-                                                     'Organizer to, as you would imagine, organize projects. '
-                                                     'A dashboard is a collection node that serves as the root of '
-                                                     'Project Organizer collections. Every user will always have '
-                                                     'one Dashboard')
+    url = LinksField({'html': 'get_absolute_url'})
+    children = HyperlinkedIdentityFieldWithMeta(view_name='nodes:node-children', lookup_field='pk', lookup_url_kwarg='node_id', meta={'count': 'get_node_count'}, link_type='related')
```

Comment:
I'm not one to be a stickler on line lengths, but let's try to keep them under 110 characters.